docs/cloud/rahti2/rahti-what-is.md

```diff
@@ -1,6 +1,13 @@
---8<-- "rahti2_announcement.md"
 ## What is Rahti 2?
 
+!!! info "Prerequisites"
+    
+    We strongly recommend that you look at the external documentation.
+    
+    Before using Rahti, you need to familiarise yourself with containers.
+    
+    You can find more information by following this [link](ext_docs.md)
+
```

Comment:
I suggest the following text: 

!!! info "Recommnedation"

    Before using Rahti, we recommend you to get familiarised with containers.

    You can find more information by following this [link](ext_docs.md)